Parse/Internal/File/State/PFFileState.m

```diff
@@ -78,13 +78,11 @@ - (NSString *)secureURLString {
     }
 
     NSString *scheme = components.scheme;
-    if (![scheme isEqualToString:@"http"]) {
-        return self.urlString;
-    }
-
-    if ([components.host isEqualToString:_PFFileStateSecureDomain]) {
+    if ([scheme isEqualToString:@"http"] || [components.host isEqualToString:_PFFileStateSecureDomain]) {
```

Comment:
I am confused and I think some clarification:

The name of the function is secureURLString, should it always return the "https" version of the self.urlString ?